dapps/example/truffle.js

```diff
@@ -1,22 +1,9 @@
 module.exports = {
-  build: {
-    "index.html": "index.html",
-    "app.js": [
-      "javascripts/app.js"
-    ],
-    "app.css": [
-      "stylesheets/app.css"
-    ],
-    "images/": "images/"
-  },
   networks: {
-      "ropsten": {
-          host: "parity-ropsten",
```

Comment:
sure. I must of have blown that away when i copied in all the latest "truffle init" app source.